src/frontend/packages/core/src/features/endpoints/create-endpoint/create-endpoint-cf-step-1/create-endpoint-cf-step-1.component.ts

```diff
@@ -151,4 +172,12 @@ export class CreateEndpointCfStep1Component implements IStepperStep, AfterConten
   toggleAdvancedOptions() {
     this.showAdvancedOptions = !this.showAdvancedOptions;
   }
+
+  toggleOverwriteEndpoints() {
```

Comment:
I've split it out into a superclass with the exception of`toggleOverwriteEndpoints`, since I would also need to move the `FormGroup` and all connected logic too.